### PR TITLE
Added Revolv

### PR DESCRIPTION
**Pull request by sgpearse**

### CODE REVIEW

File: graveyard.json [hunk 1]
```diff
@@ -19,8 +27,8 @@
     "dateClose": "2015-12-21",
     "dateOpen": "2005-09-14",
     "description": "Google Blog Search API was a way to search blogs utilizing Google.",
-    "link": "https://en.wikipedia.org/wiki/Google_Blog_Search",
-    "name": "Google Blog Search API",
+    "link": "https://revolv.com/",
```
Comment (codyogden) [inline graveyard.json]: Looks like you edited in the wrong place.

File: graveyard.json [hunk 2]
```diff
@@ -19,8 +27,8 @@
...
+    "name": "Revolv",
```
Comment (codyogden) [inline graveyard.json]: Wrong entry edited.

File: graveyard.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "dateClose": "2016-05-15",
+    "dateOpen": "2012-05-02",
+    "description": "Revolv was a monitoring and control system that allowed users to control their connected devices from a single hub.",
+    "link": "https://en.wikipedia.org/wiki/Chromecast#Chromecast_Audio",
```
Comment (codyogden) [inline graveyard.json]: Instead of `https://revolv.com`, is there another source?

Comment (sgpearse) [reply graveyard.json]: There's no wiki on the company, but there's several news articles.  Would those suffice?

File: graveyard.json (hunk 1 above)
Comment (sgpearse) [reply graveyard.json]: Oops.  Sorry about that.

File: graveyard.json (hunk 3 above)
Comment (codyogden) [reply graveyard.json]: Yeah! That would be good. I'm trying to avoid future link rot, when possible.

**Comment by codyogden** [comment]
Thank you for contributing!!!! 🙌 🎉 💯

**Pull request merged**